travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+node_js:
+  - "8.9.4"
+
+branches:
+  only:
+    - master
+
+before_script:
+  - npm install -g @angular/cli
+
+script:
+  - ng build --prod --bh
+
+after_success:
+  - chmod +x scripts/ci/post-build.sh
```

Comment:
this will  throw an error

Reply:
you 're right I forgot to delete that line